pandas/tests/test_strings.py

```diff
@@ -1943,6 +1943,13 @@ def test_split(self):
                       [u('f'), u('g'), u('h')]])
         tm.assert_series_equal(result, exp)
 
+        # expand blank split
+        values = Series(['a b c', 'a b', '', ' '])
```

Comment:
can you also test on a single string, like the original issue. and add the issue number as a comment.